korgan/pretrial_money_invariant.py

```
@@ -0,0 +1,128 @@
+        if _PENALTY_RE.search(text):
+            if not replaced:
+                rebuilt.append(request)
+                replaced = True
+            continue
+        rebuilt.append(text)
+    if not replaced:
+        rebuilt.append(request)
+    draft.demands = rebuilt
+
+    calc_line = (
+        f"Расчёт договорной неустойки: {format_kzt(result.principal)} × {result.rate_percent:g}% × "
```

Ruff (0.16.2)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)